bool    center_flg
bool    back_flg
bool    left_flg
bool    right_flg

int16   center_angle
int16   back_angle
int16   left_angle
int16   right_angle
